docs/proposals/1199-multitenant-api-proposal/README.md

```diff
@@ -0,0 +1,250 @@
+ - This API describes Specific Request Policy (SLO)
+
+
+As such, the name of the API should convey these concepts well. The `Inference-` prefix will remain, as that is a succint & accepted term related to generative AI serving.
+
+The accompanying name should also convey some or all of the other pillars, some of the names that have been considered:
+
+- `Tenant`
+  - Pros:
+    - Resource Sharing is implicit due to evocation of the 'multitenancy' concept
+	- Identification is well understood from a term like Tenant
+  - Cons: 
+    - Due to the prevalance of the 'multitenancy' concept, the term of tenant may clash with a user-defined term of 'tenant' causing a confusing interaction
+	- Does not convey Specific Request Policy well
+
+- `Flow`
+  - Pros:
+    - A common networking term helpful in describing request traffic rate control 
+  - Cons:
+    - Doesnt well describe Identification or Resource Sharing
+
+- `Objectives` (preferred)
+  - Pros:
+    - Specific Request Policy is easily understood
+	- Resource Sharing is a natural fit
+  - Cons:
+    - Identity is not well conveyed
+
+
+### CRD spec
+
+This CRD definition is a slimmed version of InferenceModel with a name change. Example here:
+
+```golang
+type InferenceObjectives struct {
+  metav1.TypeMeta
+  metav1.ObjectMeta
+
+  Spec InferenceObjectivesSpec
+}
+
+type InferenceObjectivesSpec struct {
+  PoolRef InferenceObjectReference
+
+  Criticality *int
+}
+
+```
+
+### Other changes
+- The EPP will expose a flag to define the header key that will be used to assign InferenceObjectives to
+- The modelName rewrite functionality will be included into EPP as a core feature (also handled by header) 
+- Traffic splitting on model name will be pushed to HTTPRoute using the BBR tool to extract the modelName to the header
+
+## Phase 2
+
+### CRD spec
+```golang
+
+type InferenceObjectives struct {
+  metav1.TypeMeta
+  metav1.ObjectMeta
+
+  Spec InferenceObjectivesSpec
+}
+
+type InferenceObjectivesSpec struct {
+	// Scope: Identifier
+    ID string
+
+	// Scope: Pool binding
+    PoolRef InferenceObjectReference
+
+
+	// Scope: Policy Handles (the following are different options on how policy binding may look)
+
+	// Include core policies as a field of the object
+	Criticality *int
+	FairnessPolicy NotYetDefinedFairnessPolicy
+	SLOPolicy InferenceSLOPolicy
+
+	// or
+
+	// Create an array fields that can bind arbitrary policies (including the 'core' policies)
+	Policies []InferenceObjectReference
+
+	// or 
+
+	// one-of: (Where Policy class would contain all 'core' policies that can be configured.)
+	PolicyProfileRef InferenceObjectReference
+	PolicyProfile PolicyProfile
+}
+
+// PoolObjectReference identifies an API object within the namespace of the
+// referrer.
+type InferenceObjectReference struct {
+	// Group is the group of the referent.
+	//
+	// +optional
+	// +kubebuilder:default="inference.networking.x-k8s.io"
+	Group Group `json:"group,omitempty"`
```

Comment:
Q: are Group and Kind needed?
Aren't they fixed since always referring to an `InferencePool`?